src/Sulu/Bundle/ResourceBundle/Resources/public/js/components/filters/components/form/main.js

```diff
@@ -136,7 +136,7 @@ define(['config'], function(Config) {
                     options: {
                         el: $element,
                         fieldsUrl: typeConfig.fields,
-                        operatorsUrl: constants.operatorsUrl,
+                        operatorsUrl: constants.operatorsUrl + '?locale=' + AppConfig.getUser().locale,
```

Comment:
No, the name of the filter is translated, right? In which language? I think you are also not allowed to use the locale here.